<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClockAttrCfger</class>
 <widget class="QGroupBox" name="ClockAttrCfger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>190</height>
   </rect>
  </property>
  <property name="title">
   <string>Clock</string>
  </property>
  <property name="alignment">
   <set>Qt::AlignCenter</set>
  </property>
  <property name="flat">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="dialStyleTag">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Style:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="minuteHandSize">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>110</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="tsnPointSize">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="secondHandStyle">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>150</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="hourHandTag">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hour Hand:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="minuteHandTag">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Minute Hand:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="secondHandSize">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>150</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTimeEdit" name="deltTime">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>30</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string notr="true">HH:mm:ss</string>
   </property>
  </widget>
  <widget class="QLabel" name="secondHandTag">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Second Hand:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="minuteHandStyle">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="hourHandSize">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>70</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="tsnPointTag">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Dot Style:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="tsnPointStyle">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="hourHandStyle">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="minutePointSize">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="timeDiff">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Jet Lag:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="hourPointSize">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="deltType">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>30</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="dialStyle">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="minutePointTag">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Minute Dot:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="tsnPointTag_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>3/6/9 Dot:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="hourPointTag">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hour Dot:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="ColorComboBox" name="tsnPointColor">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="ColorComboBox" name="hourPointColor">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="ColorComboBox" name="minutePointColor">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="ColorComboBox" name="secondHandColor">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>150</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="ColorComboBox" name="hourHandColor">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>70</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="ColorComboBox" name="minuteHandColor">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>110</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorComboBox</class>
   <extends>QComboBox</extends>
   <header>ColorComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
